Resources/Prototypes/_RMC14/Entities/Objects/Medical/surgical_case.yml

```diff
@@ -33,34 +44,3 @@
     - id: CMScalpel
     - id: CMHemostat
     - id: CMRetractor
-
-- type: entity
-  parent: CMSurgicalCase
-  id: CMSurgicalCaseFilledAllTools
```

Comment:
You can add a migration to migrations.yml to update all maps and fix pr'd maps to use the new entity
`Resources/migration.yml`